features/admin.application-roles.v1/components/application-roles-list.tsx

```diff
@@ -112,11 +112,11 @@ export const ApplicationRolesList = (props: ApplicationRolesListProps): ReactEle
                     ) }
                     image={ getEmptyPlaceholderIllustrations().newList }
                     imageSize="tiny"
-                    title={ t("extensions:console.applicationRoles.roleList.placeholder.title") }
+                    title={ t("applicationRoles:roleList.placeholder.title") }
```

Comment:
Let's update the copyright headers. Check other files as well.

Reply:
Fixed with [bc94c44](https://github.com/wso2/identity-apps/pull/7320/commits/bc94c4408a42f327103abcaa9f085e720fc4b53c)